app/hookSW.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
**Note:** In the future, we will want to write this file in es2015 and add it to the transpilation workflow like the files in the `js/` folder. It will be better if we want to make a es5 build one day. This file needs to be linted as well.
Ping @azasypkin.